ember-phone-input/src/components/phone-input.ts

```diff
@@ -29,10 +43,70 @@ import '../styles/styles.css';
   @public
 */
 
-export default class PhoneInputComponent extends Component {
-  @service phoneInput;
+export interface PhoneInputArgs {
+  country: string;
+  number: string | null;
+  disabled: boolean;
+  required: boolean;
+  autocomplete: string | null;
+  allowDropdown: boolean;
+  autoPlaceholder: 'aggressive' | 'off' | 'polite' | undefined;
+  customPlaceholder: string | null;
+  initialCountry: string;
+  onlyCountries: string[];
+  preferredCountries: string[];
+  separateDialCode: boolean;
+  update: (number: string | null, meta: MetaData) => void;
```

Comment:
Would the following work here?

```suggestion
  update: (number?: string, meta: MetaData) => void;
```

I feel that `number?` is more aligned with the "TypeScript way" of specifying nullable values. Same explanation applies to the related comments above

Reply:
I'd say the answer is the same as https://github.com/qonto/ember-phone-input/pull/749/files#r1374385425